common/config.go

```diff
@@ -32,76 +30,86 @@ type MoneroNode struct {
 
 // Config contains constants that are defaults for various environments
 type Config struct {
+	Env                      Environment
```

Comment:
I added this, so code can just pass the config around without passing both the config and the env. We no longer modify the configs globally, so you have to pass the config around now.